sig/generated/controllers/admin/announcements_controller.rbs

```diff
@@ -3,19 +3,20 @@ class Admin::AnnouncementsController < AdminController
 
   @announcement: Announcement
 
-  def index: () -> void
+  def index: () -> untyped
```

Comment:
I did not migrate the return types of controller actions because they'll never called directly from our code.

Reply:
When the type is 'untyped,' it's hard to tell if it's unsupported or not.
I don't think there's any issue with having a type defined, even for methods that aren't called from outside, so I believe it would be fine to define it as void. What do you think? :eyes:

Reply:
I'm either is okay.

Anyway, I think it's not a blocker of merging. I'll work on it after merging if we can agree.

Reply:
> I think it's not a blocker of merging.

I think that's absolutely correct. It would be great if you could address it in a separate PR. :pray: